critiquebrainz/frontend/templates/base.html

```diff
@@ -9,18 +9,12 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" href="{{ url_for('static', filename='img/favicon.png') }}" type="image/x-icon"/>
-
-    {% if config.COMPILE_LESS %}
```

Comment:
Don't forget to remove this variable from config file, since it's no longer used.